dags/test_dag.py

```diff
@@ -0,0 +1,53 @@
+"""DAG running in response to a AWS S3 bucket change."""
```

Comment:
we should run this as part of the builds in https://github.com/libero/elife-style-content-adapter-prototype/blob/master/.travis/smoke-test.sh
I'm not sure what is the waiting condition to understand it has finished. Perhaps move the `BashOperator` at the end and make it create a file inside a mounted volume, so that the host can check it appears?

Reply:
Having a look to see if there is something here we can leverage: https://airflow.apache.org/api.html

Reply:
How about these? :
```
curl -X GET localhost:8080/api/experimental/test

{"status":"OK"}
```
After running `libero_test_dag`:
```
curl -X GET localhost:8080/api/experimental/dags/libero_test_dag/dag_runs

[{"dag_id":"libero_test_dag",
   "dag_run_url":"/admin/airflow/graph?dag_id=libero_test_dag&execution_date=2019-05-07+15%3A01%3A51%2B00%3A00",
   "execution_date":"2019-05-07T15:01:51+00:00",
   "id":1,
   "run_id":"manual__2019-05-07T15:01:51+00:00",
   "start_date":"2019-05-07T15:01:51.962440+00:00",
   "state":"success"}]
```

Reply:
:+1: it's a published API so very stable

Reply:
I've added the status check for now. In the next PR I will add DAG tests.